src/Components/LandingPage/LandingPage.js

```diff
@@ -13,81 +20,110 @@ import {
           >
-            <Button
-              variant="plain"
-              aria-label="About image builder"
-              className="pf-u-pl-sm header-button"
-            >
-              <HelpIcon />
-            </Button>
-          </Popover>
-          <Popover
-            headerContent={'About open source'}
-            bodyContent={
-              <TextContent>
-                <Text>
-                  This service is open source, so all of its code is
-                  inspectable. Explore repositories to view and contribute to
-                  the source code.
-                </Text>
-                <Button
-                  component="a"
-                  target="_blank"
-                  variant="link"
-                  icon={<ExternalLinkAltIcon />}
-                  iconPosition="right"
-                  isInline
-                  href={
-                    'https://www.osbuild.org/guides/image-builder-service/architecture.html'
-                  }
-                >
-                  Repositories
-                </Button>
-              </TextContent>
+            <HelpIcon />
+          </Button>
+        </Popover>
+        <Popover
+          headerContent={'About open source'}
+          bodyContent={
+            <TextContent>
+              <Text>
+                This service is open source, so all of its code is inspectable.
+                Explore repositories to view and contribute to the source code.
+              </Text>
+              <Button
+                component="a"
+                target="_blank"
+                variant="link"
+                icon={<ExternalLinkAltIcon />}
+                iconPosition="right"
+                isInline
+                href={
+                  'https://www.osbuild.org/guides/image-builder-service/architecture.html'
+                }
+              >
+                Repositories
+              </Button>
+            </TextContent>
+          }
+        >
+          <Button
+            variant="plain"
+            aria-label="About Open Services"
+            className="pf-u-pl-sm header-button"
+          >
+            <CodeBranchIcon />
+          </Button>
+        </Popover>
+      </PageHeader>
+      <section className="pf-l-page__main-section pf-c-page__main-section">
+        {!isBeta() && showBetaAlert && (
+          <Alert
+            className="pf-u-mb-xl"
+            isInline
+            variant="default"
+            title="Build and launch AWS or Azure systems with custom content now available in our Beta environment."
+            actionClose={
+              <AlertActionCloseButton onClose={() => setShowBetaAlert(false)} />
+            }
+            actionLinks={
+              <Button
+                isInline
+                component="a"
+                variant="link"
+                href="/beta/insights/image-builder/landing"
+              >
+                <BetaLabel />
+                Go to Beta
```

Comment:
Update to:
Enter beta environment

Reply:
<img width="957" alt="Screenshot 2023-04-13 at 3 18 11 PM" src="https://user-images.githubusercontent.com/88330245/231861111-435ba439-48cb-4e2f-8da2-295430618fc4.png">

Example ^

Reply:
No label?

Reply:
Yes, remove the label :)

Reply:
But keep the label in the launch dialog?